### PR TITLE
Feature/boxv2

### PR DESCRIPTION
**Pull request by vonabarak**
# Description

Bot template completely reworked to use boxv2 framework

# Checklist

- [x] I've generated a new bot from the async-box template and checked that everything works:
  - [x] Linters and tests have passed
  - [x] Bot answers on `/help` command
  - [x] Bot suggests available commands
- [x] I've written a changelog for PR change
- [ ] I'll make a release when PR is approved

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 0.19.0 (Oct 8, 2021)
+
+### Changed
+
+* **Breaking change:** Template completly changed for using boxv2 bot framework.
```
Comment (lxmnk) [inline CHANGELOG.md]: Completely?

File: {{cookiecutter.bot_name}}/app/commands.py [hunk 2]
```diff
@@ -49,17 +41,3 @@ async def show_help(message: Message, bot: Bot) -> None:
...
-    """Show git commit SHA."""
-    await bot.answer_message(environ.get("GIT_COMMIT_SHA", "<undefined>"), message)
-
-
-@collector.hidden(command="/_history")
-async def history(message: Message, bot: Bot) -> None:
-    """Show history of unhandled messages."""
-    records = await Record.all()
-    text = "\n".join(row.record_data for row in records)
-    await bot.answer_message(text, message)
```
Comment (lxmnk) [inline {{cookiecutter.bot_name}}/app/commands.py:65]: Maybe leave these two commands?

File: {{cookiecutter.bot_name}}/app/resources/templates/chat_created.txt.mako [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Вас приветствует ${ bot_name }!
+
+Для более подробной информации нажмите кнопку `/help`
```
Comment (lxmnk) [inline {{cookiecutter.bot_name}}/app/resources/templates/chat_created.txt.mako]: Newline?

**Review by lxmnk — APPROVED** [review]

**Review by Dnomin — APPROVED** [review]

File: {{cookiecutter.bot_name}}/app/main.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+"""Main module."""
+
+from boxv2 import get_application  # pragma: no cover
+
+app = get_application()  # pragma: no cover
```
Comment (lxmnk) [inline {{cookiecutter.bot_name}}/app/main.py]: I suggest removing `pragma` in this file.

File: {{cookiecutter.bot_name}}/setup.cfg [hunk 5]
```diff
@@ -9,17 +7,12 @@ addopts =
     --cov=tests
     --no-cov-on-fail
     --cov-report term-missing
-    --cov-fail-under={{cookiecutter.test_cov}}
+    --cov-fail-under=0
```
Comment (lxmnk) [inline {{cookiecutter.bot_name}}/setup.cfg]: Maybe just drop this option?

Comment (vonabarak) [reply {{cookiecutter.bot_name}}/setup.cfg]: done

File: {{cookiecutter.bot_name}}/app/main.py (hunk 4 above)
Comment (vonabarak) [reply {{cookiecutter.bot_name}}/app/main.py]: done

File: {{cookiecutter.bot_name}}/app/resources/templates/chat_created.txt.mako (hunk 3 above)
Comment (vonabarak) [reply {{cookiecutter.bot_name}}/app/resources/templates/chat_created.txt.mako]: done

File: CHANGELOG.md (hunk 1 above)
Comment (vonabarak) [reply CHANGELOG.md]: fxd

**Pull request merged**